### ISSUE TITLE
arangoimport cannot find a file in the same directory

### ISSUE THREAD

**Issue by enjoysmath** [arangodb/arangodb#13321]
Windows 10, 64-bit.

Here's a command prompt session.

```
C:\Users\FruitfulApproach\Desktop\Quiver\quiver>arangoimport --file C:/Users/FruitfulApproach/Desktop/Quiver/quiver/DemoData/airports.csv --collection airports --create-collection true --type csv
Please specify a password:
Connected to ArangoDB 'http+tcp://127.0.0.1:8529, version: 3.7.5, database: '_system', username: 'root'
----------------------------------------
database:               _system
collection:             airports
create:                 yes
create database:        no
source filename:        C:/Users/FruitfulApproach/Desktop/Quiver/quiver/DemoData/airports.csv
file type:              csv
quote:                  "
separator:
threads:                2
connect timeout:        5
request timeout:        1200
----------------------------------------
Starting CSV import...

created:          0
warnings/errors:  0
updated/replaced: 0
ignored:          0
lines read:       0
2021-01-04T16:31:26Z [11628] ERROR [46995] error message(s):
2021-01-04T16:31:26Z [11628] ERROR [25049] No such file or directory

C:\Users\FruitfulApproach\Desktop\Quiver\quiver>dir
 Volume in drive C is Windows
 Volume Serial Number is E48C-105C

 Directory of C:\Users\FruitfulApproach\Desktop\Quiver\quiver

01/04/2021  08:24 AM    <DIR>          .
01/04/2021  08:24 AM    <DIR>          ..
01/03/2021  04:19 PM                12 .gitignore
01/03/2021  04:21 PM    <DIR>          .vscode
01/04/2021  08:24 AM    <DIR>          DemoData
01/04/2021  08:23 AM         5,972,303 GraphCourse_DemoData_ArangoDB-2.zip
01/03/2021  04:19 PM             1,063 LICENSE
01/03/2021  04:19 PM             4,018 Makefile
01/03/2021  06:32 PM    <DIR>          nodeArangoWebApp
01/04/2021  07:42 AM    <DIR>          node_modules
01/04/2021  07:42 AM            40,971 package-lock.json
01/04/2021  07:42 AM                79 package.json
01/03/2021  04:19 PM             7,847 README.md
01/03/2021  04:19 PM    <DIR>          screenshots
01/04/2021  08:00 AM    <DIR>          src
               7 File(s)      6,026,293 bytes
               8 Dir(s)  94,399,741,952 bytes free

C:\Users\FruitfulApproach\Desktop\Quiver\quiver>
```

This is from the very first graph db course listed here:

https://www.arangodb.com/learn/graphs/

**Comment by dothebart** [comment]
hm, I think that arangoimport doesn't handle using `/` as path separator the right way. If you use `\` in the pathname, that should work.

**Comment by maxkernbach** [comment]
Currently you can't use forward slashes `/` in the Windows command prompt (cmd), but this will be supported in 3.8.

**Comment by maxkernbach** [comment]
A backport of this fix will already be available in v3.7.7.

[Changelog](https://github.com/arangodb/arangodb/blob/5ca78591679f7ba238013cb47745200d3ea7ae3a/CHANGELOG#L4):

>   Normalize user-provided input/output directory names in arangoimport,
>   arangoexport, arangodump and arangorestore before splitting them into path
>   components, in the sense that now both forward and backward slashes can be
>   used on Windows, even interchangingly.
>

**Comment by Simran-B** [comment]
Hi @enjoysmath,
v3.7.7 with above quality-of-life improvement is now available and I successfully tested that forward and backward slashes work in cmd.exe.
